test/testshared/directives.go

```diff
@@ -82,6 +88,14 @@ func ParseTestDirectives(tb testing.TB, sourcePath string) *RunContext {
 			rc.ExpectedLinter = after
 			continue
 
+		case "//golangcitest:expected_exitcode":
+			require.NotEmpty(tb, after)
+			val, err := strconv.Atoi(after)
+			require.NoError(tb, err)
+
+			rc.ExitCode = val
+			continue
```

Comment:
Are you foreseeing any other exit codes than 0 from this directive? Right now it seems like we only set this to 0 but I wonder if it would make more sense to have a setting for tests without assertions and marking them as successful. I'm thinking something like `//golangcitest:has_no_issues` and if so set `rc.ExitCode = exitcodes.Success`?

Reply:
In fact, I think I will add tests about the `run.exitcode` but in another PR with other tests.
If you agree I would prefer to keep the explicit value.

Reply:
Sure, let's keep it! I just don't see an obvious case when a test from `testdata` would exist either successfully or with issues found. Testing other ways of exiting is a great plus but I don't think the easiest way to do that is to use this directive with another value.